### PR TITLE
✏️ update footer with links and logo

### PR DESCRIPTION
**Pull request by chris-s-friedman**
adds links to common pages and the github repo. Also points to the site logo

### CODE REVIEW

File: site/src/components/footer.js [hunk 1]
```diff
@@ -8,24 +8,24 @@ const Footer = () => (
...
               </a>
             </div>
           </div>
           <div className="tablet:grid-col"></div>
           <div className="tablet:grid-col">
             <ul className="add-list-reset">
-              <li>Footer link</li>
-              <li>Footer link</li>
-              <li>Footer link</li>
+              <li><a href="/should-i-test">Get Tested</a></li>
```
Comment (chris-s-friedman) [inline site/src/components/footer.js]: this should probably match whatever call to action we choose in #26

**Comment by dankolbman** [comment]
[Here's](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/release-2.6.0/components/preview/layout--landing.html) a demo page from the USWDS with a nice footer. It may be more fitting than what the gatsby template provided.
I feel like those footer links should be formatted a bit differently, there may be something about it in the uswds docs.

**Comment by chris-s-friedman** [comment]
for reference: https://designsystem.digital.gov/components/footer/

**Comment by chris-s-friedman** [comment]
I'll redo it like the medium/small footer styling on the uswds site

**Comment by chris-s-friedman** [comment]
image

**Pull request merged**